jenkins/flow/pr/edx-platform-bok-choy-pr.groovy

```diff
@@ -3,48 +3,50 @@ import hudson.model.*
       toolbox.slurpArtifacts(bok_choy_1)
     },
     {
-      bok_choy_2 = build('edx-platform-test-subset', sha1: sha1, SHARD: "2", TEST_SUITE: "bok-choy", PARENT_BUILD: "PR Build #" + build.number)
+      bok_choy_2 = build(subsetJob, sha1: sha1, SHARD: "2", TEST_SUITE: "bok-choy", PARENT_BUILD: "PR Build #" + build.number)
       toolbox.slurpArtifacts(bok_choy_2)
     },
     {
-      bok_choy_3 = build('edx-platform-test-subset', sha1: sha1, SHARD: "3", TEST_SUITE: "bok-choy", PARENT_BUILD: "PR Build #" + build.number)
+      bok_choy_3 = build(subsetJob, sha1: sha1, SHARD: "3", TEST_SUITE: "bok-choy", PARENT_BUILD: "PR Build #" + build.number)
       toolbox.slurpArtifacts(bok_choy_3)
     },
     {
-      bok_choy_4 = build('edx-platform-test-subset', sha1: sha1, SHARD: "4", TEST_SUITE: "bok-choy", PARENT_BUILD: "PR Build #" + build.number)
+      bok_choy_4 = build(subsetJob, sha1: sha1, SHARD: "4", TEST_SUITE: "bok-choy", PARENT_BUILD: "PR Build #" + build.number)
       toolbox.slurpArtifacts(bok_choy_4)
     },
     {
-      bok_choy_5 = build('edx-platform-test-subset', sha1: sha1, SHARD: "5", TEST_SUITE: "bok-choy", PARENT_BUILD: "PR Build #" + build.number)
+      bok_choy_5 = build(subsetJob, sha1: sha1, SHARD: "5", TEST_SUITE: "bok-choy", PARENT_BUILD: "PR Build #" + build.number)
       toolbox.slurpArtifacts(bok_choy_5)
     },
     {
-      bok_choy_6 = build('edx-platform-test-subset', sha1: sha1, SHARD: "6", TEST_SUITE: "bok-choy", PARENT_BUILD: "PR Build #" + build.number)
+      bok_choy_6 = build(subsetJob, sha1: sha1, SHARD: "6", TEST_SUITE: "bok-choy", PARENT_BUILD: "PR Build #" + build.number)
       toolbox.slurpArtifacts(bok_choy_6)
     },
     {
-      bok_choy_7 = build('edx-platform-test-subset', sha1: sha1, SHARD: "7", TEST_SUITE: "bok-choy", PARENT_BUILD: "PR Build #" + build.number)
+      bok_choy_7 = build(subsetJob, sha1: sha1, SHARD: "7", TEST_SUITE: "bok-choy", PARENT_BUILD: "PR Build #" + build.number)
       toolbox.slurpArtifacts(bok_choy_7)
     },
     {
-      bok_choy_8 = build('edx-platform-test-subset', sha1: sha1, SHARD: "8", TEST_SUITE: "bok-choy", PARENT_BUILD: "PR Build #" + build.number)
+      bok_choy_8 = build(subsetJob, sha1: sha1, SHARD: "8", TEST_SUITE: "bok-choy", PARENT_BUILD: "PR Build #" + build.number)
       toolbox.slurpArtifacts(bok_choy_8)
     },
     {
-      bok_choy_9 = build('edx-platform-test-subset', sha1: sha1, SHARD: "9", TEST_SUITE: "bok-choy", PARENT_BUILD: "PR Build #" + build.number)
+      bok_choy_9 = build(subsetJob, sha1: sha1, SHARD: "9", TEST_SUITE: "bok-choy", PARENT_BUILD: "PR Build #" + build.number)
       toolbox.slurpArtifacts(bok_choy_9)
     },
   )
 }rescue{
   FilePath artifactsDir =  new FilePath(build.artifactManager.getArtifactsDir())
-  FilePath copyToDir = new FilePath(build.workspace, "edx-platform")
+  FilePath copyToDir = new FilePath(build.workspace, repoName)
```

Comment:
Note that this will be the subdirectory on the git clone for the workspace. I think you'll need a change on [this line](https://github.com/edx/jenkins-job-dsl/pull/19/files#diff-f390436c4fc56fa9265b55ebacd15c39L117) on your other PR for the artifacts to be properly copied & archived in the parent job.

For purposes of what we're trying to achieve right now, I don't think you need to change this line. 

Or, alternatively, you could make the change in both places.

Screenshot to give extra context. This directory is where these are being copied into.
![image](https://cloud.githubusercontent.com/assets/2004678/16660578/f4a4c72c-443c-11e6-807b-103fb045235f.png)